client/src/components/audit-row/resolver.ts

```diff
@@ -0,0 +1,103 @@
+import { Audit } from '../../types';
+
+type Data = { [key: string]: never };
+
+interface IResolver {
+  (params: Data, query: Data, payload: Data): string;
+}
+
+const resolveAddIPToAllowList: IResolver = (
+  _params: Data,
+  _query: Data,
+  payload: Data,
+) => `Added IP ${payload.ip} to the AllowList`;
+
+const resolveDeleteIPfromAllowList: IResolver = () =>
+  `Deleted IP from the AllowList.`;
+
+const resolveCreateNewStaff: IResolver = (
+  _params: Data,
+  _query: Data,
+  payload: Data,
+) => `Added new staff with email: ${payload.email}.`;
+
+const resolveUpdateVariant: IResolver = (params: Data) =>
+  `Updated variant with ID of ${params.variantId} on product with ID of ${params.productId}`;
+
+const resolveDeleteProduct: IResolver = (params: Data) =>
+  `Deleted product with ID of ${params.id}.`;
```

Comment:
I realized that it would be much more useful to the user to know which object they actually deleted. But this is something we can consider as a follow up